example-play/app/controllers/HmrcPlayInterpreter.scala

```diff
@@ -177,7 +177,7 @@ case class HmrcPlayInterpreter(
           tag("main")(cls:= "govuk-main-wrapper ", id:="main-content", role:="main")(
             div(id:="mainBody") (
               breadcrumbs.drop(1).headOption.map{ back =>
-                a (href:=back.mkString("/"), cls:="govuk-back-link")(messages({back :+ "back"}.mkString(".")))
```

Comment:
Will we have to make changes to our backlinks based on this?

Reply:
no, this is just in the example play application.